docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+nightscout:
+  build: .
+  links:
+   - database
+   - broker
+  ports:
+   - "1337:1337"
+  environment:
+   - MONGO_CONNECTION=mongodb://database/nightscout
+   - API_SECRET=mylittlesecret
```

Comment:
Yes, that should not be a problem. The user should adjust it's environment variables such as the secret and the Pebble keys. The only visible ports from the outside are the exposed ports, in the current case the HTTP port of Node and the MQTT port.

Docker uses containers to isolate the application from the big-bad outside world. For example the MongoDB container is referred by `database` and is linked to the Node container. It does not expose any ports to the outside world, so it cannot be reached from outside of the docker ecosystem.

The environment variables shown in the .yml are passed on to the container as environment variables. I will dive in to see if there is any way to make these easy configurable.